common/src/main/java/com/wynntils/gui/widgets/TextInputBoxWidget.java

```diff
@@ -254,7 +254,7 @@ public boolean isFocused() {
     }
 
     @Override
-    public void updateWidgetNarration(NarrationElementOutput narrationElementOutput) {}
+    protected void updateWidgetNarration(NarrationElementOutput narrationElementOutput) {}
```

Comment:
The method is now protected. We should change all public cases to protected.

Reply:
We have this in lots of other places, which are public. I suggest we revert these two odd ones, and then make a separate change to make them protected. (And it is always legal to widen scope, so it is more an issue of style)